### PR TITLE
Restructure micropython libs and adding print statements to functions in the microbit library that we will not implement in the simulator

### PR DESCRIPTION
**Pull request by vandyliu**
PLEASE TEST IT OUT

# Description:
[AB#34028](https://microsoftgarage.visualstudio.com/002806e2-ebaa-4672-9d2e-5fe5d29154ef/_workitems/edit/34028)
- Reorganized all micropython libraries under `src/micropython`
- Adding print statements to functions in the microbit library that we will not implement in the simulator
- In Express.py (for CPX)
- In compass.py, i2c.py, spi.py and microbit_model.py (for microbit)
- In audio, radio, music, speech, neopixel, utime
- print statement looks like: `{function_name} on line {line_number} in {user_file_name} is not implemented in the simulator but it will work on the actual device!`

## Type of change

Please delete options that are not relevant.
- [ ] New feature (non-breaking change which adds functionality)

# Limitations:

Please describe limitations of this PR

# Testing:

Tested in the simulator with:
```
from microbit import *

spi.init()
spi.read(2)
spi.write(2)
spi.write_readinto(1, 2)
reset()
panic(23)
i2c.init()
i2c.read(2, 3)
i2c.write(2, 3)
i2c.scan()
compass.calibrate()
compass.clear_calibration()
compass.get_field_strength()
compass.get_x()
compass.get_y()
compass.get_z()
compass.heading()
compass.is_calibrated()
```
```
import microbit

microbit.spi.init()
microbit.spi.read(2)
microbit.spi.write(2)
microbit.spi.write_readinto(1, 2)
microbit.reset()
microbit.panic(23)
microbit.i2c.init()
microbit.i2c.read(2, 3)
microbit.i2c.write(2, 3)
microbit.i2c.scan()
microbit.compass.calibrate()
microbit.compass.clear_calibration()
microbit.compass.get_field_strength()
microbit.compass.get_x()
microbit.compass.get_y()
microbit.compass.get_z()
microbit.compass.heading()
microbit.compass.is_calibrated()
```
```
from adafruit_circuitplayground import cp

cp.play_tone(2, 2)
cp.start_tone(2)
cp.stop_tone()
cp.adjust_touch_threshold(3)
cp.tapped()
```
```
import audio
audio.play("a")
audio.AudioFrame()

import music
music.set_tempo()
music.get_tempo()
music.play("a")
music.pitch("a")
music.stop()

import neopixel
n = neopixel.NeoPixel("a", "1")
n.clear()
n.show()

import radio
radio.on()
radio.off()
radio.config()
radio.reset()
radio.send_bytes("S")
radio.receive_bytes()
radio.receive_bytes_into("a")
radio.send("?")
radio.receive()
radio.receive_full()

import speech
speech.translate("A")
speech.pronounce("asa")
speech.say("a")
speech.sing("a")

import utime
utime.ticks_ms()
utime.ticks_us()
utime.ticks_add(1, 1)
utime.ticks_diff(1,1)
```
- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] My code has been formatted with `npm run format` and passes the checks in `npm run check`
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by nasadigital — APPROVED** [review]
LGTM!

**Review by andreamah — APPROVED** [review]
Yay! Awesome 😁

File: src/adafruit_circuitplayground/express.py [hunk 1]
```diff
@@ -154,12 +154,12 @@ def touch_A6(self):
...
         return self.__touch(7)
 
-    def adjust_touch_threshold(self, adjustement):
+    def adjust_touch_threshold(self, adjustment):
         """Not implemented!
         The CPX Simulator doesn't use capacitive touch threshold.
         """
         telemetry_py.send_telemetry(TelemetryEvent.CPX_API_ADJUST_THRESHOLD)
-        raise NotImplementedError(CONSTANTS.NOT_IMPLEMENTED_ERROR)
+        utils.print_for_unimplemented_functions(Express.adjust_touch_threshold.__name__)
```
Comment (xnkevinnguyen) [inline src/adafruit_circuitplayground/express.py:162]: Nice move to print with the function definition name :)

File: src/adafruit_circuitplayground/express.py [hunk 2]
```diff
@@ -86,7 +86,7 @@ def tapped(self):
         """  Not Implemented!
         """
         telemetry_py.send_telemetry(TelemetryEvent.CPX_API_TAPPED)
-        raise NotImplementedError(CONSTANTS.NOT_IMPLEMENTED_ERROR)
+        utils.print_for_unimplemented_functions("tapped")
```
Comment (xnkevinnguyen) [inline src/adafruit_circuitplayground/express.py:89]: Can you do the same for this function? (use __name)

**Review by xnkevinnguyen — CHANGES_REQUESTED** [review]

File: src/adafruit_circuitplayground/express.py (hunk 2 above)
Comment (vandyliu) [reply src/adafruit_circuitplayground/express.py:89]: I cannot because it is a @property ):

**Review by xnkevinnguyen — APPROVED** [review]

**Review by nasadigital — APPROVED** [review]

**Review by andreamah — APPROVED** [review]
(❁´◡`❁) good to go!!

**Pull request merged**